### PR TITLE
New version: athlabs.Spyglass version 23.11.1

### PR DESCRIPTION
**Pull request by tianyeeT**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/158497)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\VALIDA~1\AppData\Local\Temp\MicrosoftEdgeWebview2Setup.exe returned exit code: -1
> Executable C:\Program Files (x86)\Microsoft\EdgeUpdate\1.3.181.5\MicrosoftEdgeUpdate.exe returned exit code: -1606220282
> Executable C:\Program Files (x86)\Microsoft\EdgeUpdate\1.3.181.5\MicrosoftEdgeUpdateCore.exe returned exit code: -2147024894
> Executable C:\Program Files (x86)\Microsoft\EdgeUpdate\MicrosoftEdgeUpdate.exe returned exit code: -1606220282
> Executable C:\Program Files (x86)\Microsoft\EdgeUpdate\1.3.181.5\MicrosoftEdgeUpdateSetup.exe returned exit code: -1
> Executable C:\Program Files (x86)\Microsoft\EdgeUpdate\Download\{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5}\126.0.2592.61\MicrosoftEdge_X64_126.0.2592.61.exe returned exit code: -1
> Executable C:\Program Files (x86)\Microsoft\EdgeCore\126.0.2592.61\BHO\ie_to_edge_stub.exe returned exit code: 1
> Executable C:\Program Files (x86)\Microsoft\EdgeCore\126.0.2592.61\cookie_exporter.exe returned exit code: -1610612722
> Executable C:\Program Files (x86)\Microsoft\EdgeCore\126.0.2592.61\elevation_service.exe returned exit code: 1063
> Executable C:\Program Files (x86)\Microsoft\EdgeCore\126.0.2592.61\msedgewebview2.exe returned exit code: 13
> Executable C:\Program Files (x86)\Microsoft\EdgeCore\126.0.2592.61\msedge_pwa_launcher.exe returned exit code: -2147483645
> Executable C:\Program Files (x86)\Microsoft\EdgeCore\126.0.2592.61\pwahelper.exe returned exit code: 1
> Executable C:\Program Files (x86)\Microsoft\EdgeCore\126.0.2592.61\Installer\setup.exe returned exit code: 3
> Executable C:\Program Files (x86)\Microsoft\EdgeWebView\Application\126.0.2592.61\cookie_exporter.exe returned exit code: -1610612722
> Executable C:\Program Files (x86)\Microsoft\EdgeWebView\Application\126.0.2592.61\elevation_service.exe returned exit code: 1063
> Executable C:\Program Files (x86)\Microsoft\EdgeWebView\Application\126.0.2592.61\msedge.exe returned exit code: 13
> Executable C:\Program Files (x86)\Microsoft\EdgeWebView\Application\126.0.2592.61\msedgewebview2.exe returned exit code: 13
> Executable C:\Program Files (x86)\Microsoft\EdgeWebView\Application\126.0.2592.61\msedge_pwa_launcher.exe returned exit code: -2147483645
> Executable C:\Program Files (x86)\Microsoft\EdgeWebView\Application\126.0.2592.61\pwahelper.exe returned exit code: 1
> Executable C:\Program Files (x86)\Microsoft\EdgeWebView\Application\126.0.2592.61\Installer\setup.exe returned exit code: 501
> Executable C:\Program Files (x86)\Microsoft\EdgeWebView\Application\126.0.2592.61\BHO\ie_to_edge_stub.exe returned exit code: 1
> Executable C:\Users\validator\AppData\Local\Spyglass\Spyglass.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\Spyglass\pdftotext.exe returned exit code: 99
> Executable C:\Users\validator\AppData\Local\Spyglass\spyglass-debug.exe returned exit code: -1073741515
> Executable C:\Users\validator\AppData\Local\Spyglass\spyglass-server.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

Automated error analysis suggests 1606 may mean `ERROR_UNKNOWN_FEATURE` - Feature ID not registered.

Automated error analysis suggests 1610 may mean `ERROR_BAD_CONFIGURATION` - The configuration data for this product is corrupt.

(Automated response - build 886.)

File: manifests/a/athlabs/Spyglass/23.11.1/athlabs.Spyglass.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+InstallerLocale: en-US
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2023-11-06
+Installers:
```
Comment (stephengillie) [inline manifests/a/athlabs/Spyglass/23.11.1/athlabs.Spyglass.installer.yaml:15]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Comment by stephengillie** [comment]
This package installs and launches normally on a Windows 10 VM.

**Comment by stephengillie** [comment]
This package installs and launches normally on a Windows 10 VM.

**Review by stephengillie — APPROVED** [review]

**Comment by tianyeeT** [comment]
Thanks! @stephengillie

**Comment by stephengillie** [comment]
- https://github.com/microsoft/winget-pkgs/issues/142854
-

**Pull request merged**